tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -6025,12 +6021,17 @@ def globalReadGuardKBody(tP):
                   #   numElementsPerLoad = 2
                   # elif self.states.archCaps["HasEccHalf"]:
                   #   destVgprHi = self.vgprPool.checkOut(1, 'destVgprHi')
+                  if (not tP["isM"]) and kernel["DirectToLds%c"%tc] and (not tP["tlu"]) and tP["glvw"]>=4 and kernel["AssertSummationElementMultiple"] % 4 == 0:
```

Comment:
So far, no 1 or 2 Byte Global Read with DTL.
We need to enable 4 byte load in TailLoop for now.